unityfiles/Assets/Scripts/Helper/consts.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+public enum STATE {
```

Comment:
Tror jeg ville gått for GameState i såfall :D Litt mer beskrivende når man bruker :D Mye kan ha STATE 🌮 🐟

Reply:
Valgte spesifikt STATE siden å ha en separat for LevelStates og GameStates kann fort lede til forvirrig og blandig.

Reply:
Eller, siden jeg spesifiserer type på state kan det kansje funke bra å splitte den i to